ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/ChatRequestMessage.qml

```diff
@@ -18,9 +18,12 @@ Item {
 
     StyledText {
         id: contactText1
-        text: qsTr("You need to be mutual contacts with this person for them to receive your messages")
-//        text: !isContact ? qsTr("You need to be mutual contacts with this person for them to receive your messages") :
-//                           qsTr("Waiting for %1 to accept your request").arg(Utils.removeStatusEns(chatsModel.channelView.activeChannel.name))
+        //% "You need to be mutual contacts with this person for them to receive your messages"
+        //% "You need to be mutual contacts with this person for them to receive your messages"
+        //% "Waiting for %1 to accept your request"
+        text: qsTrId("you-need-to-be-mutual-contacts-with-this-person-for-them-to-receive-your-messages")
+//        text: !isContact ? qsTrId("you-need-to-be-mutual-contacts-with-this-person-for-them-to-receive-your-messages") :
+//                           qsTrId("waiting-for--1-to-accept-your-request").arg(Utils.removeStatusEns(chatsModel.channelView.activeChannel.name))
```

Comment:
The script messed up here because of the comments. Could you move the comments to the right place, thanks.